pom.xml

```diff
@@ -67,14 +69,31 @@
         <dependency>
             <groupId>org.jvnet.hudson.plugins</groupId>
             <artifactId>analysis-core</artifactId>
-            <version>1.71</version>
-            <type>hpi</type>
+            <version>1.73-beta-SNAPSHOT</version><!-- TODO: change to 1.73 before release -->
```

Comment:
Great!

FYI [the fix in analysis-core](https://github.com/jenkinsci/analysis-core-plugin/commit/34c2aef5018ec716c7a6b6ba160eea429bcd7755)